src/hooks/useDailyXvs.ts

```diff
@@ -0,0 +1,36 @@
+import { useMemo } from 'react';
+import { BigNumber } from 'bignumber.js';
+import { useGetVTokenDailyXvs } from 'clients/api';
+import { convertWeiToCoins } from 'utilities/common';
+import { Asset } from 'types';
+import { XVS_TOKEN_ID } from 'constants/xvs';
+
+export const useDailyXvs = ({
+  accountAddress,
+  assets,
+}: {
+  accountAddress?: string;
+  assets: Asset[];
+}) => {
+  const { data: vTokenDailyXvsWei, isLoading } = useGetVTokenDailyXvs(
+    { accountAddress: accountAddress || '' },
+    { enabled: !!accountAddress },
+  );
+
+  const { dailyXvsDistributionInterestsCents } = useMemo(() => {
+    const dailyXvsTokens = convertWeiToCoins({
+      valueWei: vTokenDailyXvsWei || new BigNumber(0),
```

Comment:
While `vTokenDailyXvsWei` is `undefined`, I think we should return `dailyXvsDistributionInterestsCents` as `undefined` too, this way we're consistent with our logic.